internal/handlers/common/params.go

```diff
@@ -242,8 +243,24 @@ func addNumbers(v1, v2 any) (any, error) {
 		case float64:
 			return v2 + float64(v1), nil
 		case int32:
+			if v2 == math.MaxInt32 && v1 > 0 {
+				return int64(v1) + int64(v2), nil
+			}
+
+			if v2 == math.MinInt32 && v1 < 0 {
+				return int64(v1) + int64(v2), nil
+			}
+
 			return v1 + v2, nil
 		case int64:
+			if v1 > 0 && v2 == math.MaxInt64 {
+				return nil, errLongExceeded
+			}
```

Comment:
What if `v2` is equal to `MaxInt64 - 1` and `v1` is equal to at least 2?

Reply:
Good catch! We should check what MongoDB does in that case.